vanillajs/cards/app.js

```diff
@@ -0,0 +1,25 @@
+const placeholderInfo = {
+  image: "https://via.placeholder.com/150/771796",
+  name: "Jane Doe",
+  title: "VP Engineering"
+};
+
+class Card {
+  constructor(container) {
+    this.container = document.querySelector(container);
+  }
+
+  init() {
+    const card = `
+    <div class="card">
```

Comment:
You can run very easily into collisions if you name your class 'card'. The solution? let the user provide a class name and have a default ready in case this value is empty.